test/current_toolchain_tools/BUILD.bazel

```diff
@@ -0,0 +1,43 @@
+# Executable targets will output a pattern similar to the following
+# cargo 1.53.0 (4369396ce 2021-04-27)
+# Also Note, rustc_srcs is too big for this test
+_FILES = {
```

Comment:
Nit: rename this directory from `current_toolchain_tools` to `current_toolchain_files` to match the `toolchain_files` rule name?

Reply:
Done